vault.js

```diff
@@ -1,4 +1,19 @@
 'use strict';
 module.exports = function() {
-  
+    let vaultKey = '';
```

Comment:
So this would only allow one thing to be in the "vault" at any point in time. You might want to use a object here instead so that you could hold multiple "things" in your vault.